src/Graphics/Canvas.js

```diff
@@ -1,6 +1,8 @@
 /* global exports */
 "use strict";
 
+const unit = {};
```

Comment:
My point was that you don't need to return _anything_ in particular (or at all). Just omitting the `return` statement will be fine.

Reply:
Omitted. I would normally favor the explicit returns since they documents intent.